### ISSUE TITLE
.comm symbols are not necessarily in the same order in the source and object codes

### ISSUE THREAD

**Issue by monniaux** [AbsInt/CompCert-small-tests#1]
Commit d7e6f65f3ca89e6de411dd63177ee12280a04649 introduces the following test
```
int ta[1], tb[1];

char * test11(void)
{
  int *p = ta;
  while(p != tb) p++;
  return "OK";
}
```

This test relies on `ta` and `tb` being laid next to each other in memory, `tb` after `ta`.
On the KVX platform with Kalray's `kvx-cos-gcc`, `tb` comes after `ta`. The ordering actually depends on the names of the symbols (if instead `patata` and `patatb` are used, then `patata` comes before `patatb`).

I'm disabling this test for the KVX platform. Just to let you know it may bite you for other development chains.

**Comment by monniaux** [comment]
Here is a fix:
https://github.com/AbsInt/CompCert-small-tests/pull/2

(Renaming `ta` and `tb` into `patati` and `patata` works on KVX but not on MacOS X. I looked for a more robust approach.)

**Comment by xavierleroy** [comment]
Fixed by #2.
